src/main/java/uk/gov/hmcts/reform/professionalapi/controller/WelcomeController.java

```diff
@@ -40,7 +40,7 @@
      *
      * @return Welcome message from the service.
      */
-    @ApiOperation("Welcome message for the System Reference Data API")
+    @ApiOperation("Welcome message for the Professional Reference Data API")
```

Comment:
Please check if the WelcomeControllerTest (unit test) needs to be updated

Reply:
Not required @AdilOoze, as it is swagger specific message

Reply:
Thank folks. Let me move this shortly.